packages/utils/package.json

```diff
@@ -56,6 +56,10 @@
         "ow": "^0.28.1",
         "robots-parser": "^3.0.1",
         "sax": "^1.3.0",
-        "tslib": "^2.4.0"
+        "tslib": "^2.4.0",
+        "whatwg-mimetype": "^4.0.0"
+    },
+    "devDependencies": {
+        "@types/whatwg-mimetype": "^3.0.2"
```

Comment:
The version mismatch is concerning, but it looks OK after a superficial inspection :shrug: